docs/modules/ROOT/pages/how-tos/configuring/prefetching-dependencies.adoc

```diff
@@ -389,6 +389,8 @@ In {ProductName}, from the *Applications* view, select the application build you
 * Select the *Logs* tab.
 * Alternatively, you can click *build-container*. When the right panel opens, select the *Logs* tab to see a partial view of the log for that build.
 
+NOTE: Konflux also supports prefetching RPM content which requires a Red Hat subscription. For more information see xref:./activation-keys-subscription.adoc[Using Red Hat activation keys to access subscription content].
```

Comment:
Nitpick: it'd be slightly better if we also added the right content anchor to the link (I'm pretty sure my following suggestion works, but I haven't tested it)

```suggestion
NOTE: Konflux also supports prefetching RPM content which requires a Red Hat subscription. For more information see xref:./activation-keys-subscription.adoc#hermetic-network-isolated-builds[Using Red Hat activation keys to access subscription content].
```

Reply:
It worked - added!